delta-api/src/main/java/io/cdap/delta/api/DDLEvent.java

```diff
@@ -28,14 +28,14 @@
  * A DDL change event.
  */
 public class DDLEvent extends ChangeEvent {
-  private final DDLOperation operation;
+  private final DDLOperation.Type operation;
   private final Schema schema;
   private final String database;
   private final String prevTable;
   private final String table;
```

Comment:
if the table name is in the Operation, it shouldn't also be in the event.

Reply:
yeah noticed it earlier.. will remove it from `DDLEvent` and `DMLEvent` classes as a followup PR.